pkg/querier/queryrange/roundtrip.go

```diff
@@ -6,19 +6,19 @@ import (
 	"strings"
 	"time"
 
+	"github.com/cortexproject/cortex/pkg/tenant"
+
 	"github.com/go-kit/log"
 	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/prometheus/model/labels"
 	"github.com/weaveworks/common/httpgrpc"
 
-	"github.com/grafana/loki/pkg/tenant"
-
 	"github.com/grafana/loki/pkg/loghttp"
 	"github.com/grafana/loki/pkg/logql"
 	"github.com/grafana/loki/pkg/querier/queryrange/queryrangebase"
 	"github.com/grafana/loki/pkg/storage/chunk"
-	"github.com/grafana/loki/pkg/storage/chunk/cache"
+	lokicache "github.com/grafana/loki/pkg/storage/chunk/cache"
```

Comment:
```suggestion
	"github.com/grafana/loki/pkg/storage/chunk/cache"
```
Now that we only have one imported cache pkg, we should be able to drop the alias.